### DISCUSSION TITLE
Time out error when running ./bootstrap-local-environment.sh using latest-experimental version for adding payments

### DISCUSSION THREAD

**Discussion by emanuel-skai** [NillionNetwork/discussions#76] [Q&A] [answered]
Hi,

I'm following the steps described in [python-quickstart](https://nillion-fe-docs.vercel.app/python-quickstart) for using the latest experimental version, which adds payments for storage and compute operations in the Python Client.

After 2.5 minutes of running the ./bootstrap-local-environment.sh script, I encountered a timeout error as shown below:
```
ℹ️ found bin NILLION_DEVNET     -> [nillion-devnet]
ℹ️ found bin NILLION_CLI        -> [nillion]
--------------------
Updating your .env files with nillion-devnet environment info... This may take a minute.
--------------------
Timeout reached while waiting for cluster to be ready in '/var/folders/tc/9cs8xygx3kl08b4v4pch_0h00000gn/T/tmp.1BNLPx2W9n'
```
I tried running it with RUST_LOG="DEBUG", but the script seems to run indefinitely. The logs it produces in the initial 10 minutes of execution are attached in the zip file.

Any guidance on this would be highly appreciated.

[nillion_devnet_logs.txt.zip](https://github.com/user-attachments/files/16076809/nillion_devnet_logs.txt.zip)

**Comment by Davetbutler** [comment, marked as answer]
Hey. Yes, for the new `latest-experimental` version, [you will need to use this branch of the python quickstart](https://github.com/NillionNetwork/nillion-python-starter/tree/update-quickstart-july-release). The bootstrap script is no longer needed as we write configs automatically to an `.env` file when `nillion-devnet` is run.

However, note that soon (today - about 5 hours from now), we will be releasing this version publicly (`latest`) which means it will be the `main` branch of the quickstart.

[The new python examples repo is already up to date for the new changes](https://github.com/NillionNetwork/python-examples)

I think this will solve your issues :)

Edit: I now see the source of the confusion, the vercel quickstart docs were not updated, only the notion page, as things are going public so soon, it will all be documented in there.

**Comment by emanuel-skai** [comment]
Thanks a lot, @Davetbutler. My confusion was indeed due to the Vercel Quickstart docs not being updated. The issue is solved now.
